fe/fe-core/pom.xml

```diff
@@ -641,7 +641,21 @@ under the License.
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.30</version>
```

Comment:
Why not use env in $FE_HOME/pom.xml to control the version uniformly.

Reply:
thank you, i resolved it.